atoma-bin/atoma_node.rs

```diff
@@ -129,9 +129,9 @@ async fn initialize_tokenizers(
                     revision.clone(),
                 ));
 
-                let tokenizer_filename = repo
-                    .get("tokenizer.json")
-                    .expect("Failed to get tokenizer.json");
```

Comment:
why not `expect(&format!("Failed to get tokenizer.json for model {model}"))` ?

Reply:
Totally agree !

Reply:
Take a look at: https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call

Reply:
I added the printing of the error message, explicitly